pkg/asset/ignition/bootstrap/gcp/storage.go

```diff
@@ -79,9 +67,10 @@ func CreateStorage(ctx context.Context, ic *installconfig.InstallConfig, bucketH
 // CreateSignedURL creates a signed url and correlates the signed url with a storage bucket.
 func CreateSignedURL(clusterID string) (string, error) {
 	bucketName := GetBootstrapStorageName(clusterID)
-	handle, err := CreateBucketHandle(context.Background(), bucketName)
+
+	client, err := NewStorageClient(context.Background())
```

Comment:
nit: I would probably prefer this to be context.TODO rather than background. As `CreateSignedURL` needs a context, ideally we would extend it to accept a context from the caller--but I'm not sure if that caller has a context. Context.TODO to me suggests we would like to plumb this into the top-level context but haven't gotten around to it. This is very much a nit. :P